dll/keyboard/kbdbe/kbdbe.c

```diff
@@ -98,9 +98,6 @@ ROSDATA USHORT scancode_to_vk[] = {
   /* - 77 - */
   VK_EMPTY, VK_EMPTY, VK_EMPTY, VK_EMPTY,
   VK_EMPTY, VK_EMPTY, VK_EMPTY, VK_EMPTY, /* PA1 */
-  VK_EMPTY,
```

Comment:
It's different from the previous layouts?

Reply:
Yes, there are some differences in different layouts. Some had multiple zeros in array, some only in the end.